tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
-envlist = py{27,35,36},docs,public_names
+envlist = py{27,35,36},lint,docs,public_names
 skip_missing_interpreters = true
-toxworkdir = {toxinidir}/build/test/tox
```

Comment:
Is everyone cool with this going back to `.tox`? I'm the one that originally set it to `build/test/tox`, as part of a hope that we could keep all build-time artifacts confined to a single directory, but as things have evolved (particularly with how Pip-installable plugins are built), that's probably not practical to fully realize.

I think it now might be best to let Tox go in its default place, since that's going to be the least surprising to other developers and tooling.